packages/functional-tests/tests/subscription-tests/coupons-ui-tests/ui-tests.spec.ts

```diff
@@ -7,22 +7,18 @@ import { MetricsObserver } from '../../../lib/metrics';
 
 test.describe('severity-2 #smoke', () => {
   test.describe('ui functionality', () => {
-    let metricsObserver: MetricsObserver;
-
-    test.beforeEach(({ pages: { subscribe } }) => {
-      test.slow();
-      metricsObserver = new MetricsObserver(subscribe);
-      metricsObserver.startTracking();
-    });
-
     test('verify plan change funnel metrics & coupon feature not available when changing plans', async ({
-      page,
       pages: { relier, subscribe },
     }, { project }) => {
       test.skip(
         project.name === 'production',
         'no real payment method available in prod'
       );
+      test.slow();
```

Comment:
Did we just move this from the `beforeEach()` hook because it was the only test in the suite?

Reply:
Yep that's right!